pyart/graph/gridmapdisplay.py

```diff
@@ -76,8 +76,8 @@ def __init__(self, grid, debug=False):
         # set up the projection
         lat0 = grid.origin_latitude['data'][0]
         lon0 = grid.origin_longitude['data'][0]
-        self.proj = pyproj.Proj(proj='aeqd', datum='NAD83',
-                                lat_0=lat0, lon_0=lon0)
+        self.proj = pyproj.Proj(
```

Comment:
Yes, thanks for getting another set of eyes on this.  I came up with a few other changes in addition that should go into this PR.